src/Simulation/Core/TypeExtensions.cs

```diff
@@ -144,6 +146,38 @@ public static Type[] GetTupleFieldTypes(this Type arg)
             }
         }
 
+        /// <summary>
+        /// Given a <see cref="Type"/> and its arguments, format its non-qubit arguments into a string.
+        /// Returns null if no arguments found.
+        /// </summary>
+        public static string? ArgsToString(this Type t, object args)
```

Comment:
wondering if this can/should be incorporated to enhance the debugger proxy.

Reply:
What ideas do you have regarding this?